release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -518,12 +518,20 @@ With this release, you can install a cluster on Nutanix by using the named, prel
 For more information, see xref:../installing/installing_nutanix/installation-config-parameters-nutanix.adoc#installation-configuration-parameters-additional-nutanix_installation-config-parameters-nutanix[Additional Nutanix configuration parameters].
 
 [id="ocp-4-18-installation-and-update-nutanix-multiple-nics_{context}"]
-==== Installing a cluster on Nutanix with up to 32 subnets
-With this release, Nutanix supports more than one subnet for the Prism Element where you deployed an {product-title} cluster to. A maximum of 32 subnets for each Prism Element is supported.
+==== Installing a cluster on Nutanix with multiple subnets
+With this release, you can install a Nutanix cluster with more than one subnet for the Prism Element into which you are deploying an {product-title} cluster. 
 
 For more information, see xref:../installing/installing_nutanix/installing-nutanix-installer-provisioned.adoc#installation-configuring-nutanix-failure-domains_installing-nutanix-installer-provisioned[Configuring failure domains] and xref:../installing/installing_nutanix/installation-config-parameters-nutanix.adoc#installation-configuration-parameters-additional-nutanix_installation-config-parameters-nutanix[Additional Nutanix configuration parameters].
 
-For an existing Nutanix cluster, you can add multiple subnets by using machine sets. For more information, see xref:../installing/installing_nutanix/nutanix-failure-domains.adoc#post-installation-configuring-nutanix-failure-domains_nutanix-failure-domains[Adding failure domains to the Infrastructure CR].
+For an existing Nutanix cluster, you can add multiple subnets by using xref:../machine_management/creating_machinesets/creating-machineset-nutanix.adoc#machineset-yaml-nutanix_creating-machineset-nutanix[compute] or xref:../machine_management/control_plane_machine_management/cpmso_provider_configurations/cpmso-config-options-nutanix.adoc#cpmso-yaml-provider-spec-nutanix_cpmso-config-options-nutanix[control plane] machine sets.
+
+[id="ocp-4-18-installation-and-update-vsphere-multiple-nics_{context}"]
+==== Installing a cluster on {vmw-full} with multiple network interface controllers (Technology Preview)
```

Comment:
@snarayan-redhat I could swear we talked about rel notes for this (and that I am responsible for them) but I cannot find the comment to that effect. Lmk if I missed something, and if these seem okay for you for the install side :bow: